src/constants/styles.ts

```diff
@@ -0,0 +1,10 @@
+//navigationBar styles
+export const navigationBackgroundStyle =
+  'fixed left-0 bottom-0 w-background flex justify-center';
+export const navigationConetentStyle =
+  'flex w-content h-navigationBar bg-purple-500';
+
+export const navigationButtonStyle = 'flex-1 flex flex-col h-full min-h-0 pb-4';
+export const navigationButtonImageStyle =
+  'flex-[8_0_0%] min-h-0 object-contain';
+export const navigationButtonTextStyle = 'flex-1 min-h-0';
```

Comment:
```suggestion
export const NAVIGATION_BACKGROUND =
  'fixed left-0 bottom-0 w-background flex justify-center';
export const NAVIGATION_CONTENT =
  'flex w-content h-navigationBar bg-purple-500';

export const NAVIGATION_BUTTON = 'flex-1 flex flex-col h-full min-h-0 pb-4';
export const NAVIGATION_BUTTON_IMAGE =
  'flex-[8_0_0%] min-h-0 object-contain';
export const NAVIGATION_BUTTON_TEXT = 'flex-1 min-h-0';
```
이런식으로 상수니까 상수 컨벤션을 지키는게 좋지 않을까요?